src/main/java/sirius/biz/storage/layer3/MutableVirtualFile.java

```diff
@@ -34,19 +37,66 @@
      * Provides a prediacte which is always <tt>false</tt>.
      */
     public static final Predicate<VirtualFile> CONSTANT_FALSE = ignored -> false;
+    private static final String REGEX_ILLEGAL_FILE_CHARS = "[/\\\\?%*:|\"<>]";
```

Comment:
Add javadoc here and explain here these characters came from (e.g. link to https://en.wikipedia.org/wiki/Filename)